src/plugins/graph/components/graph.tsx

```diff
@@ -53,6 +54,9 @@ export const Graph = observer(function Graph({graphController, graphRef}: IProps
   useGraphModel({dotsRef, graphModel, enableAnimation, instanceId});
 
   useEffect(function setupPlotArea() {
+    // TODO: Replace this quick and dirty method for setting attributes
+    dataset && graphModel.setAttributeID('x', dataset.attributes[0].id);
+    dataset && graphModel.setAttributeID('y', dataset.attributes[1].id);
```

Comment:
This is the right hack for configuring the graph to show particular attributes on the axes. There may be a better place to implement the hack, but this seems fine for now.

Reply:
This should move so that it occurs at the moment of linking rather than in the graph component itself.